graphql/schema/scalars.graphql

```diff
@@ -2,4 +2,5 @@
 scalar Time
 scalar Duration
 scalar StringMap
+scalar BooleanMap
```

Comment:
added a boolean map here so the slack options could be resolved without having to write a custom resolver